vscode/build.mjs

```diff
@@ -78,12 +77,13 @@ export function copyKatex(destDir, useLightTheme) {
   );
 
   // Also copy the GitHub markdown CSS
-  const cssFileName = useLightTheme
-    ? "github-markdown-light.css"
-    : "github-markdown.css";
   copyFileSync(
-    join(libsDir, `github-markdown-css/${cssFileName}`),
-    join(katexDest, "github-markdown.css"),
+    join(libsDir, `github-markdown-css/github-markdown-light.css`),
+    join(katexDest, "github-markdown-light.css"),
+  );
+  copyFileSync(
+    join(libsDir, `github-markdown-css/github-markdown-dark.css`),
+    join(katexDest, "github-markdown-dark.css"),
```

Comment:
It is intended. It's part of the change I mentioned with switching the style sheet based on the VS Code theme (rather than using the one style sheet which uses the OS theme).